### ISSUE TITLE
[BUG] Updating to the latest version in HACS gives an import error

### ISSUE THREAD

**Issue by kneirinck** [openmotics/home-assistant#37]
Updating to the latest version through HACS is giving an import error because of an out of sync pyhaopenmotics.
I fixed it by manually uninstall pyhaopenmotics and letting it redownload it.
But I think the actual fix would be to bump pyhaopenmotics to 0.0.3 and point to that version.
I haven't had time yet to verify that or create PRs but just wanted to report in case anyone encounters this issue.

**Comment by tmds** [comment]
> I fixed it by manually uninstall pyhaopenmotics and letting it redownload it.

@kneirinck what are the steps to do this?

**Comment by woutercoppens** [comment]
‘ha core rebuild’ will also do the job.

**Comment by kneirinck** [comment]
> > I fixed it by manually uninstall pyhaopenmotics and letting it redownload it.
> 
> @kneirinck what are the steps to do this?

IIRC run `pip uninstall pyhaopenmotics` on your HA docker instance or HAOS and then restart HA

**Comment by tmds** [comment]
I run hass in a container.

I tried the `ha` command but it wasn't available.

The `pip uninstall` didn't work for me either: it didn't find the package. Maybe I was running it in the wrong folder.

I found the `pyhaopenmotics` directory and overwrote it with https://github.com/openmotics/pyhaopenmotics/tree/main/src/pyhaopenmotics. Not the cleanest way to go about it... but it's up and running again now.

I ran into this issue by doing an upgrade through hacs.
Will it also occur on a fresh install?

What needs to happen to solve this?
